src/core/annotation.js

```diff
@@ -34,6 +34,10 @@ import {
 import { Catalog, FileSpec, ObjectLoader } from "./obj.js";
 import { collectActions, getInheritableProperty } from "./core_utils.js";
 import { Dict, isDict, isName, isRef, isStream, Name } from "./primitives.js";
+import {
+  dumpDefaultAppearance,
+  parseDefaultAppearance,
+} from "./defaultappearance.js";
```

Comment:
Aside from the already added comments that I agree with, we should call this file `default_appearance.js`.